backend/pom.xml

```diff
@@ -158,6 +163,11 @@
       <artifactId>mongodb</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.oracle.database.jdbc</groupId>
+      <artifactId>ojdbc8</artifactId>
+      <version>19.10.0.0</version>
```

Comment:
Please use `21.1.0.0` to be compliant with the driver used by Debezium upstream project.  Also, this dependency should be marked as `provided` which should address your concern about POM dependencies since we need to compile against it but cannot distribute it.